e2e_demo/scala/cleanup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+rm -rf sparkdemoproject-3 spark-* *.jar *.tgz *.gz hadoop-*
+rm -rf ../../pipelinecompare/warehouse/*
+rm -f /tmp/spark-migration-jars/*
+cd sparkdemoproject && gradle clean
```

Comment:
Why delete the tgz and spark dirs each run?
Also gradle clean does not make sense for sbt build

Reply:
I'm not sure of the best way to cleanup all the stuff in sbt. Somehow `sbt clean` still leaves the target directory intact for me. Is that supposed to happen? @holdenk 

Anyway, moving this gradle-specific thing to the other script

Reply:
> Why delete the tgz and spark dirs each run?

Thought we anyway download it every time? So just wanted the dir to look fresh when I was tracking the execution prompt-by-prompt. Not good?

Reply:
No, the `! -f` is if file does not exist so we only download the files if they are not present (and they should not change so downloading them each time is wasteful).